.github/workflows/inductor-perf-test-nightly.yml

```diff
@@ -9,7 +9,7 @@ on:
   workflow_dispatch:
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'push' }}
```

Comment:
I don't think you need to remove `${{ github.event_name == 'workflow_dispatch' }}` here, as it would still handle the path where the workflow is invoked manually.  

On the other hand, you can have both checks like what other workflows like pull does https://github.com/pytorch/pytorch/blob/master/.github/workflows/pull.yml#L16, i.e.

```
${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'push' }}-${{ github.event_name == 'schedule' }}
```

Reply:
adding on to this, trunk, periodic, and other ciflow label triggered workflows have a concurrency group that cancels on new commit pushed for PRs

Reply:
Oh, that's the `${{ github.ref_type == 'branch' && github.sha }}` part I guess

Reply:
Thank both of you for the helpful suggestions. Please help review again, thanks!